test-data/unit/parse-errors.test

```diff
@@ -287,17 +287,6 @@ def g(*x): # type: (X) -> Y
 file:1: error: Inconsistent use of '*' in function signature
 file:3: error: Inconsistent use of '*' in function signature
 
-[case testCommentFunctionAnnotationVarArgMispatch2]
```

Comment:
Removed this test because I made `**kwargs` happening before `*args` an error no matter what.

```
>>> def foo(**kwargs, *args):
  File "<stdin>", line 1
    def foo(**kwargs, *args):
```